src/AddTodo/AddTodo.js

```diff
@@ -0,0 +1,30 @@
+
+const AddTodo = ({item, setItem, handleAddTask}) => {
+
+    const handleTaskChange = (e)=> {
+        setItem(
+            {
+                ...item,
+                title : e.target.value
+            }
+        )
+    }
+
+    const handleSubmit = (e) =>{
+        e.preventDefault();
+        handleAddTask(item);
+    }
+    return(
+        <>
+            <form onSubmit= {handleSubmit}>
+                <label>
+                    Task:
+                    <input type = "text" value = {item.title} onChange = {handleTaskChange}/>
+                </label>
+                <input type="submit" value="Save"/>
+            </form>
+        </>
```

Comment:
If we are returning a single node we don't need to wrap it in the fragment.
```suggestion
        <form onSubmit= {handleSubmit}>
            <label>
                Task:
                <input type = "text" value = {item.title} onChange = {handleTaskChange}/>
            </label>
            <input type="submit" value="Save"/>
        </form>
```